toonz/sources/include/tfx.h

```diff
@@ -210,16 +210,16 @@ class DVAPI TFxTimeRegion {
   double m_end;
 };
 
-inline DVAPI TFxTimeRegion operator+(const TFxTimeRegion &tr1,
-                                     const TFxTimeRegion &tr2) {
+inline TFxTimeRegion operator+(const TFxTimeRegion &tr1,
+                               const TFxTimeRegion &tr2) {
   return TFxTimeRegion(tr1) += tr2;
 }
 
-inline DVAPI TFxTimeRegion operator+(const TFxTimeRegion &tr1, double shift) {
+inline TFxTimeRegion operator+(const TFxTimeRegion &tr1, double shift) {
```

Comment:
I've removed DVAPI for inline function only. There is no reason to use dllexport/dllimport for inline functions. Because they are inline.